### DISCUSSION TITLE
How to Avoid Similar Auto Translation

### DISCUSSION THREAD

**Discussion by wbotelhos** [fnando/i18n-js#718] [Q&A] [answered]
With the following pattern:

```yml
- '{pt-BR,en,ru}.edit'
```

The following text are translated:

```
"edit_item": "Editar %{item}",
"editor": {
  "other": "Editores"
}
```

I can use the following to avoid it:

```yml
- '!{pt-BR,en,ru}.edit_item'
- '!{pt-BR,en,ru}.editor'
```

But shouldn't it be the default? I'm not using the wildcard in the final like `'{pt-BR,en,ru}.edit*'`.

What am I missing? Thanks!

**Comment by fnando** [comment]
I think you found a bug. I'll take a look at it.

**Reply by fnando** [reply, marked as answer]
Fixed here: https://github.com/fnando/glob/pull/21

**Reply by fnando** [reply]
Just released glob v0.4.1 with this fix.

**Reply by wbotelhos** [reply]
Thank you! ❤️
